tool/jt.rb

```diff
@@ -172,9 +172,7 @@ def self.find_jruby
     if ENV['RUBY_BIN']
       ENV['RUBY_BIN']
     elsif mx?
-      "#{JRUBY_DIR}/tool/jruby_mx"
-    else
-      "#{JRUBY_DIR}/bin/jruby"
+      "#{JRUBY_DIR}/bin/truffleruby"
```

Comment:
What I've done is have a `$home/bin/truffleruby` that will always link to the real launcher in any distribution.